pokemongo_bot/inventory.py

```diff
@@ -1415,6 +1499,13 @@ def items():
     """
     return _inventory.items
 
+def applied_items():
```

Comment:
Yes, but here I'm not interested in the items in the bag. I want to get the items (and time left on those items) that are in use. Am I doing it wrong?